tests/components/simplefin/test_coordinator.py

```diff
@@ -0,0 +1,52 @@
+"""Test the data update coordinator - hopefully in a HA approved manner."""
```

Comment:
please don't test the internals (coordinator), but rather add tests what happens if an error occurs (e.g. sensor going unavailable)

Reply:
ok removed it